index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>GOLD GYM</title>
+  <!-- Fontawesome cdn -->
+  <script src="https://kit.fontawesome.com/27219c429c.js" crossorigin="anonymous"></script>
+  <!-- Main css style -->
+  <link rel="stylesheet" href="source/style/style.css" />
+</head>
+
+<body>
+  <!-- Navigation Menu Section -->
+  <nav class="nav">
+    <a href="index.html" class="logo"><img src="source/images/logo.svg" alt="GOLD GYM logo"></a>
+    <!-- fa-solid fa-x -->
+    <button id="show-menu"><i class="fas fa-bars" id="change-icon"></i></button>
+    <!-- nav menu -->
+    <ul class="nav-menu" id="nav-menu">
+      <li class="nav-item"><a href="index.html" class="nav-link" id="link-active">Home</a></li>
+      <li class="nav-item"><a href="about-us.html" class="nav-link">About</a></li>
+      <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+      <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
+      <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+    </ul>
+  </nav>
+
+  <!-- Hero Section -->
+  <header class="hero-wrapper">
+    <div class="hero-container">
+      <p class="greeting">"Hello Athlete's"</p>
+      <h1 class="hero-header-text">
+        Victory isn't defined by wins and losses, it's defined by effort
+      </h1>
+      <p class="hero-header-desc">With our team you’ll reach your goals quickly. Enjoy a spacious and well equipped free
+        weight area to meet everyone's needs with the best equipment in the industry. Equipped with the space and
+        equipment for the normal person to competitive athletes or athletic teams.
+      </p>
+      <p class="hero-header-time"><span class="workout-time"><i class="fas fa-clock"></i>24/7</span> (Monday) ~ (Friday)
+      </p>
+    </div>
+  </header>
+
+  <!-- Services Section -->
+  <section class="services">
+    <h1 class="services-header">Main Services</h1>
```

Comment:
- [ ] There should be no more than 1 `<h1>` tag on a page. Please take a look at [this article](https://www.contentkingapp.com/academy/headings/) about the headings and fix this issue.